python-threatexchange/threatexchange/hashing/pdq_faiss_matcher.py

```diff
@@ -101,6 +101,47 @@ def search(
             for i in range(len(query_vectors))
         ]
 
+    def search_with_distance_in_result(
+        self,
+        queries: t.Sequence[PDQ_HASH_TYPE],
```

Comment:
ymmv: This is a new method, you can choose to just pick a winner between string and bytes. I'm not sure, since while bytes is more accurate for PDQ specifically, all of the other signal types just generalize to str (i.e. text input). 

I think in all the calling points you are probably coming from str, but dunno

Reply:
Done